frappe/desk/form/load.py

```diff
@@ -30,7 +31,7 @@ def getdoc(doctype, name, user=None):
 	if not name:
 		name = doctype
 
-	if not frappe.db.exists(doctype, name):
+	if not frappe.db.exists(doctype, name) and not is_virtual_doctype(doctype):
```

Comment:
`is_vitrtual_doctype` is likely site cached (and thus faster than a query)? Should we reverse the order? 

Ignore if you think it doesn't matter much